test_assets/MaterialXTest/readme.md

```diff
@@ -0,0 +1,47 @@
+## USD assets with MaterialX
+
+### Enabling MaterialX in USD
+
+To enable materialx, build USD with `--materialx` option
+
+```python build_scripts\build_usd.py  --materialx ```
+
+### Sample USD files
+For details on how to use MaterialX in USD see ASWF Open Source 2021 [talk](https://materialx.org/assets/ASWF_OSD2021_MaterialX_slides_final.pdf).
+
+The MaterialX files and teapot model referenced in these samples are from ASWF MaterialX [repository](https://github.com/AcademySoftwareFoundation/MaterialX/tree/main/resources/Materials/Examples). 
+
```

Comment:
```suggestion
The MaterialX files and teapot model are TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
All rights reserved, and covered by the Apache 2.0 license.
```

Reply:
The teapot note should go into `_common/README.md` ?